server/sonar-server/src/main/java/org/sonar/server/measure/ws/ComponentTreeAction.java

```diff
@@ -152,6 +158,11 @@ public void define(WebService.NewController context) {
       .setSince("5.5")
       .setPossibleValues(1, 2, 3, 4, 5);
 
+    action.createParam(PARAM_METRIC_SORT_FILTER)
+      .setDescription("Filter out components that do not have measure on the sorted metric")
```

Comment:
The description is misleading, it could be 
- all : return all measures
- withMeasuresOnly : Filter out components that do not have measure on the sorted metric